TransformationSystem/Client/TransformationCLI.py

```diff
@@ -14,6 +14,7 @@
 
 from DIRAC.TransformationSystem.Client.Transformation           import Transformation
 from DIRAC.TransformationSystem.Client.TransformationClient     import TransformationClient
+from DIRAC.Resources.Catalog.FileCatalogClient                  import FileCatalogClient
```

Comment:
You are right. It seems i used in a previous version that i superseded.
I am leaving now office. I look your comments tomorrow morning.